code/game/gamemodes/traitor/double_agents.dm

```diff
@@ -6,12 +6,12 @@
 	name = "Sleeper Agents"

```

Comment:
"Sleeper Agents" is actually supposed to be lowercase. It was uppercase when it was called Internal Affairs so my find & replace kept it uppercase. All the other game mode names are lowercase.

If you could fix it for consistency that'd be great.